core/backend-itwin-client/src/oidc/introspection/IntrospectionClient.ts

```diff
@@ -66,7 +66,7 @@ export class IntrospectionClient {
   }
 
   public async introspect(requestContext: AuthorizedClientRequestContext): Promise<IntrospectionResponse> {
-    const accessTokenStr = requestContext.accessToken.toTokenString(IncludePrefix.No);
+    const accessTokenStr = requestContext.accessToken?.substr(requestContext.accessToken.indexOf(" ") + 1) ?? "";

```

Comment:
revisit this once we decide if token will always have prefix or not

Reply:
@pbell97 do you want to update this to use the method you made?

Reply:
@calebmshafer It is updated to use that method, this is just an outdated snippet.

Reply:
Ah my bad. Resolving...